mitosheet/src/mito/types.tsx

```diff
@@ -890,7 +890,7 @@ export interface UIState {
     selectedTabType: 'data' | 'graph';
     currOpenToolbarDropdown: undefined | ToolbarDropdowns;
     highlightedColumnIndex?: number;
-    toolbarDropdown: 'import' | 'format' | 'dtype' | 'export' | 'reset-index' | undefined;
+    toolbarDropdown: 'import' | 'format' | 'dtype' | 'export' | 'reset-index' | 'MATH' | 'LOGIC' | 'FINANCE' | 'DATE' | 'TEXT' | 'REFERENCE' | 'MORE' | undefined;
```

Comment:
For the actual toolbar dropdown type, I'd be down to make these like `formula-math` or `formula-logic`. Just because `MORE` is not super informative...